### PR TITLE
age: add Chinese translation

### PR DESCRIPTION
**Pull request by LiLittleCat**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/common/age.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# age
+
+> 一个简单、现代、安全的文件加密工具。
+> 更多信息：<https://age-encryption.org>.
+
+- 生成一个可以用口令解密的加密文件：
```
Comment (blueskyson) [inline pages.zh/common/age.md:6]:
```suggestion
- 生成一个可以用密码短语（passphrase）解密的加密文件：
```

File: pages.zh/common/age.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- 用一个或多个公钥加密一个文件，这些公钥以字面形式输入：
+
+`age --recipient {{公钥_1}} --recipient {{公钥_2}} {{路径/到/未加密文件}} --output {{路径/到/已加密文件}}`
+
+- 用收件人文件中指定的一个或多个公钥来加密一个文件：
+
+`age --recipients-file {{路径/到/收件人文件}} {{路径/到/未加密文件}} --output {{路径/到/已加密文件}}`
+
+- 用口令解密一个文件：
```
Comment (blueskyson) [inline pages.zh/common/age.md:22]:
```suggestion
- 用密码短语解密一个文件：
```

**Review by blueskyson — COMMENTED** [review]
A more commonly translation of "passphrase".

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**